src/applications/static-pages/sass/modules/_m-education-wizard.scss

```diff
@@ -49,7 +48,30 @@
 }
 
 .wizard-fieldset {
-  input[type="checkbox"], input[type="radio"] {
+  input[type='checkbox'],
+  input[type='radio'] {
     margin-left: -4rem;
   }
 }
+
+.wizard-edith-nourse-content {
+  @media (min-width: $medium-screen) {
+    margin-right: 64px;
+  }
+}
+.checkBenefitsLink {
+  white-space: nowrap;
+}
+
+.radioText {
+  display: inline-flex;
+
+  @media (min-width: $small-screen) {
+    width: 90%;
+  }
+  width: 75%;
+}
+
+label {
+  display: block !important;
+}
```

Comment:
Were the labels being displayed some other way?

Reply:
Upon inspection of the label, it looked to be inheriting the style from: 
`errorable-radio-button label` 
Which was setting `display` to `inline-block;`, and caused styling issues conflicting with the design review comments provided by our UX team.